.github/workflows/pr_checks.yml

```diff
@@ -1,5 +1,8 @@
 name: Run CI
-on: [push]
+on: [
+  push,
+  pull_request_target
```

Comment:
TL;DR from my comment (https://github.com/ethyca/fidesops/pull/86#issuecomment-972983287)

If you set this to `pull_request` instead of `pull_request_target` for now, we'll get 90% of the value of CI but I'd expect tests that rely on secrets to fail. That's a secure place to start from